Integrations/MailListener-POP3/MailListener-POP3.yml

```
@@ -0,0 +1,60 @@
+commonfields:
+  id: MailListener-POP3
+  version: -1
+name: MailListener-POP3
+display: MailListener-POP3
+category: Messaging
+description: Integration Template
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Server URL (e.g. https://example.com)
```

it's not an http server, it's pop3
```suggestion
- display: Server URL (e.g. example.com)
```